icons/index.ts

```diff
@@ -2336,6 +2340,50 @@ const ICON_LIST: IconListItem[] = [
     icon: BoxesIcon,
     keywords: ['boxes', 'box', 'container', 'package', 'shipping'],
   },
+  {
+    name: 'user-round-plus',
+    icon: UserRoundPlusIcon,
+    keywords: [
+      'user',
+      'plus',
+      'add',
+      'create',
+      'new'
+    ],
+  },
+  {
+    name: 'panel-left-open',
+    icon: PanelLeftOpenIcon,
+    keywords: [
+      'panel',
+      'left',
+      'open',
+      'menu',
+      'navigation'
+    ],
+  },
+  {
+    name: 'panel-left-close',
+    icon: PanelLeftCloseIcon,
+    keywords: [
+      'panel',
+      'left',
+      'close',
+      'menu',
+      'navigation'
+    ],
+  },
+  {
+    name: 'panel-right-open',
+    icon: PanelRightOpenIcon,
+    keywords: [
+      'panel',
+      'right',
+      'open',
+      'menu',
+      'navigation'
+    ],
+  },
```

Comment:
I was wondering at first why the icons don't appear in the local dev after I ran `gen-cli`. Turns out I still need to add the new icons manually here. Do you think we should update the gen-cli script to add the new icons to this file?

I might be able to send PR separately for it.

Reply:
no, it works as expected. you add new file with icon and then you add this icons to this array with lucide keywords and then run `gen-cli` to generate shadcn config

Reply:
Got it! So it’s part of “make your changes to the codebase”. 

![IMG_9633](https://github.com/user-attachments/assets/3eb5ecd6-741c-4f29-92fa-c41ce04ebfe4)